qiskit/visualization/pulse_visualization.py

```diff
@@ -26,39 +28,85 @@
-        channel_scales (dict[Channel, float]): channel independent scaling as a
-            dictionary of `Channel` object.
-        channels_to_plot (list): Deprecated, see `channels`
-        plot_all (bool): Plot empty channels
-        plot_range (tuple): A tuple of time range to plot
-        interactive (bool): When set true show the circuit in a new window
-            (this depends on the matplotlib backend being used supporting this)
-        table (bool): Draw event table for supported commands
-        label (bool): Label individual instructions
-        framechange (bool): Add framechange indicators
-        scaling (float): Deprecated, see `scale`
-        channels (list): A list of channel names to plot
-        show_framechange_channels (bool): Plot channels with only framechanges
+        data: Pulse or schedule object to plot.
+        dt: Time interval of samples. Pulses are visualized in the unit of
+            cycle time if not provided.
+        style: A style sheet to configure plot appearance.
+            See `qiskit.visualization.pulse.qcstyle` for more information.
+        filename: Name required to save pulse image. The drawer just returns
+            `matplot.Figure` object if not provided.
+        interp_method: Interpolation function. See example.
+            Interpolation is disabled in default.
+            See `qiskit.visualization.pulse.interpolation` for more information.
+        scale: Scaling of waveform amplitude. Pulses are automatically
+            scaled channel by channel if not provided.
+        channel_scales: Dictionary of scale factor for specific channels.
+            Scale of channels not specified here is overwritten by `scale`.
+        channels_to_plot: Deprecated, see `channels`.
+        plot_all: When set `True` plot empty channels.
+        plot_range: A tuple of time range to plot.
+        interactive: When set `True` show the circuit in a new window.
+            This depends on the matplotlib backend being used supporting this.
+        table: When set `True` draw event table for supported commands.
+        label: When set `True` draw label for individual instructions.
+        framechange: When set `True` draw framechange indicators.
+        scaling: Deprecated, see `scale`.
+        channels: A list of channel names to plot.
+            All non-empty channels are shown if not provided.
+        show_framechange_channels: When set `True` plot channels
+            with only framechange instructions.
 
     Returns:
-        matplotlib.figure: A matplotlib figure object for the pulse envelope
+        A matplotlib figure object for the pulse envelope.
+
+    Example:
+        This example shows how to customize your schedule plot.
+        Custom style sheet is applied and pulse names are added to the plot.
+        Unimportant channels are removed and the schedule is truncated to
+        draw out important pulses.
+
+        .. jupyter-execute::
```

Comment:
A newline must be added after `.. jupyter-execute::` otherwise the build will fail. (The PR test builds didn't fail yet because API Doc for this file is not generated as it hasn't been added to the `qiskit.visualization.__init__.py`)

```suggestion
        .. jupyter-execute::
        
```